.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 
 # The following owners will be the default owners for everything in the repo.
 # Unless a later match takes precedence
-@bitwarden/team-auth-dev
\ No newline at end of file
+* @bitwarden/team-auth-dev
```

Comment:
⚠️ I strongly prefer newlines at the ends of files.

Reply:
Fixed, had to turn off prettier for `codeowners` since it keeps trying to format it as markdown.

Reply:
Yes, so annoying. Needs an exception in the config.